site/write_ups/2022/02-11-22_writeup.md

```diff
@@ -0,0 +1,98 @@
+
+## Case Study: Data Collection Practises in the EU (in particular, Sweden)
+
+In Sweden, no ethnicity grouping is allowed to be recorded. The [intention](https://www.thelocal.se/20180508/why-sweden-doesnt-keep-stats-on-ethnic-background-and-crime/) is to prevent biases being built into the system. 
+
+In a more comprehensive [Analysis and comparative review of equality data collection practises in the European Union](https://ec.europa.eu/info/sites/default/files/data_collection_in_the_field_of_ethnicity.pdf), this is noted in context amongst other country:
+
+"Sweden: Race is not used as the term is considered to be closely linked to racism.
+Ethnic origin (etniskt ursprung) is more accepted, used predominantly within the
+anti-discrimination area. The politically correct term used in public documents is
+foreign origin as opposed to Swedish origin. These two terms are officially defined
+by Statistics Sweden as follows.31 Persons of Swedish origin are persons born in
+Sweden who’s both parents were born in Sweden or persons born in Sweden who’s
+one parent was born in Sweden and one parent was born outside of Sweden.
+Persons of foreign origin are persons born outside of Sweden or persons born in
+Sweden who’s both parents were born outside of Sweden."
+
+Finland agonised over inclusion and equality and talk about it openly, and have had great success. The UK has been really bad with this. UK doesn't collect data for White Other, for example. This needs to be intersectional - look at 'black women' vs 'asian women' for example. 
+
+## Saini also refers to the overlapping of different inequalities and privileges in different contexts. How does this intersectionality of identities impact the way we do or should analyse data about demographics?
+
+Intersectionality is important, but the more intersectional a group you want to look at, the more difficult the data is to get (especially in big enough numbers to apply machine learning to).
+
+We touched on this earlier, but splitting categories seems sensible (although numbers get small in increasingly narrow categories). Ethnicity recording across the National Health Service (NHS) has improved dramatically over the past decade. [This study](https://pubmed.ncbi.nlm.nih.gov/24323951/) profiles the completeness, consistency and representativeness of routinely collected ethnicity data in both primary care and hospital settings. The second greatest category in ethnicity recorded is second after White British. The question is if this group is still predominantely White British or not.
+
```

Comment:
```suggestion

As we mentioned earlier, perhaps the most useful thing to do is just to ask people what you need to know for each context at a time. If we need to know if someone is privileged or disadvantaged in a particular context regarding a particular characteristic, we should probably just ask about that instead of trying to find a catch-all measure that will never work all of the time. 
That way we can gather information that is relevant to what we need to know, and not rely on proxy values.
After all, humans are not one-dimensional, and our identities aren't either!
```